docs/pages/includes/install-linux.mdx

```diff
@@ -87,6 +89,17 @@
         ```
   </TabItem>
 
+  <TabItem label="Amazon Linux 2/RHEL Legacy (RPM)">
```

Comment:
Are we planning on maintaining this old repo? I almost think we should remove the instructions to avoid confusion (some people just won't read the "(Legacy)" in the header)

Reply:
We'll probably be maintaining old repos (both apt and rpm) for foreseeable future. We thought about removing these instructions but don't want to scare people who still depend on them. Maybe adding a more prominent warning/admonition on this tab would be appropriate - saying that these are legacy ones and you should use the other repo but these aren't going anywhere. cc @fheinecke